demos/getting_started.md

```diff
@@ -0,0 +1,146 @@
+# YesWorkflow Web Components Getting Started Guide
+
+## Prerequisites: 
+### Required:
+* Python 3.6 or greater
+* Pip for the installed Python3 environment
+* [Clone the YesWorkflow Web Components Repository](https://github.com/aniehuser/senior-design-group10)
+* Install project dependencies by executing `pip install -r requirements.txt`
+
+### Optional:
+The YesWorkflow Web Components Repository comes with an already packaged YesWorkflow Jar file that can be used for the purposes of saving one's work to the website.
+
+If one would like to see the source code for the CLI Java tool it can be found at the [YesWorkflow Prototypes Repository](https://github.com/aniehuser/yw-prototypes).
+
+For more detailed instruction on how to begin using and developing the CLI please see the README found within this repository.
+
+## Installation:
+After installing all prerequisites navigate to the folder in the cloned website repository labeled scripts. To quickly launch the website execute *deploy_python3.sh* if on MacOs or Linux. If on a Window's environment execute *deploy_python.sh*. Note that these two files are simply a collection of bash commands that will initialize and begin running the YesWorkflow Web Components website. 
+
+If unable to run these bash files, the Python3 commands inside can be executed separately through the terminal.
+
+For documentation on the commands found in these files please see the [Django documentation](https://docs.djangoproject.com/en/2.1/).
+
+## Key Terms:
+* Workflows
+* Versions
+* Runs
+* Tags
+
+## CLI Tool
+### YesWorkflow Save Command and Configurations
+The Jar file to be executed from the terminal can be found at the root folder of the YesWorkflow Web Components repository.
+
+YesWorkflow CLI's Save command works by executing and aggregating all YesWorkflow commands and then sending important data up to your deployed YesWorkflow WebComponents instance. To use, one can type in the command prompt
+
+```
+java -jar yesworkflow.jar save my_scripty.py
```

Comment:
some feedback I got was to be more clear about what `my_scripty.py` is. We should mention that it is a python file that has YesWorkflow annotations.

I also got feedback that it wasn't clear if we could you `.R` files or .`matlab` files, so that could be a nice clarifier as well